src/nwb_datajoint/common/common_behav.py

```diff
@@ -103,20 +105,30 @@ class StateScriptFile(dj.Imported):
     """
 
     def make(self, key):
+        """Add a new row to the StateScriptFile table. Requires keys "nwb_file_name", "file_object_id"."""
         nwb_file_name = key['nwb_file_name']
         nwb_file_abspath = Nwbfile.get_abs_path(nwb_file_name)
         nwbf = get_nwb_file(nwb_file_abspath)
-        associated_files = nwbf.processing.get(
-            'associated_files') or nwbf.processing.get('associated files')
-        if associated_files is not None:
-            # TODO type checking that associated_file_obj is of type ndx_franklab_novela.AssociatedFiles
-            for associated_file_obj in associated_files.data_interfaces.values():
-                # parse the task_epochs string
-                epoch_list = associated_file_obj.task_epochs.split(',')
-                # find the file associated with this epoch
-                if str(key['epoch']) in epoch_list:
-                    key['file_object_id'] = associated_file_obj.object_id
-                    self.insert1(key)
+
+        associated_files = nwbf.processing.get('associated_files') or nwbf.processing.get('associated files')
```

Comment:
just curious: I see that the associated statescript files are defined in the extension, but are they directly included in the nwb file or are they just linked?